cmd/clusterctl/client/config/reader_viper.go

```diff
@@ -33,6 +36,8 @@ const (
 	ConfigFolder = ".cluster-api"
 	// ConfigName defines the name of the config file under ConfigFolder
 	ConfigName = "clusterctl"
+	//DownloadConfigFile is the location of the config file when fetching the config from a remote location
+	DownloadConfigFile = "/tmp/clusterctl-download.yaml"
```

Comment:
Also, what about moving this file under `~/.cluster-api`, so everything clusterctl related is in one location?

Reply:
done

Reply:
did a small change, but i'm not sure about that.

Reply:
The new change lgtm to, but if this is creating troubles feel free to rollback